e2e_playwright/st_audio.py

```diff
@@ -24,3 +24,7 @@
 st.audio(url2, start_time=10, end_time=13)
 
 st.audio(url2, start_time=15, end_time=19, loop=True)
+
```

Comment:
One test case you could add here is for the autoplay prevention aspect. E.g. add this to the python code:

```suggestion
if st.button("Create some elements to unmount component"):
    for i in range(3):
        time.sleep(1)
        st.write("Another element")
```

And in e2e test, you 1) check the checkbox and make sure the audio is playing 2) click the button and wait for the rerun to finish (`wait_for_app_run`) 3) check that the autoplay is put on false on the audio + check that the audio is paused / not started

Reply:
Added the remaining tests :D